src/main/java/com/stream_pi/client/window/settings/GeneralTab.java

```diff
@@ -584,6 +621,21 @@ else if(port > 65535)
                     exceptionAndAlertHandler.handleSevereException(e);
                 }
             }
+            
+            if(!config.getCurrentAnimationName().equals(animationComboBox.getCurrentSelectedItem()))
+            {
+                syncWithServer = true;
+                
+                try
+                {
+                    config.setCurrentAnimationFullName(animationComboBox.getCurrentSelectedItem());
+                    config.save();
```

Comment:
Not a good idea to save this here because the newly changed config gets changed in the end anyways.

Reply:
done